content/chainguard/chainguard-enforce/troubleshooting-tips.md

```diff
@@ -18,33 +18,193 @@ toc: true
+```
+
+If `jwks` being inaccessible is the issue, you'll receive an error here.
+
+```
+Error from server (NotFound): the server could not find the requested resource
+```
+
+The output above reveals that the endpoints are not available to `chainctl`, whcih may be due to an auth proxy between `chainctl` and the controlplane.
+
+To solve for this issue, you'll need an OpenID Connect Well-Known Configuration Endpoint, which should be available at the `/.well-known/openid-configuration` path, This will enable you to use `chainctl` for installation and to register an identity. You can check that the file is there with `kubectl`.
+
+```sh
+kubectl get --raw /.well-known/openid-configuration
+```
+
+If you receive an error that the requested resource was not found, make sure you are using the `--private` option on installation. You can review guidance on this option in the [Configuring Enforcer Options](/chainguard/chainguard-enforce/installation/configure-enforcer-options-during-installation/#install-with-chainctl) guide.
+
+If you are using Rancher, you'll be able to generate “direct access” kubeconfigs that bypass the auth proxy and go directly to the cluster. Please follow the [Rancher documentation to create a new kubeconfig file](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/manage-clusters/access-clusters/use-kubectl-and-kubeconfig#authenticating-directly-with-a-downstream-cluster).
+
+### Installing Enforce with Rancher
+
+If you're using Rancher for Kubernetes, you'll need to point directly to your kubeconfig file:
+
+```sh
+KUBECONFIG=/etc/rancher/rke2/rke2.yaml chainctl cluster install --group=$GROUP --private
+```
+
+If you cannot reach the OIDC endpoint, this will cause issues for a cluster install, review the [section above](#cluster-oidc-configuration-exit-status) for guidance on troubleshooting this. 
+
+Additionally, you'll need direct access to the authorized endpoint and not the Rancher proxy URL. When you run `kubectl config view`, you should receive something like the output below, without the Rancher URL.
+
+```
+clusters:
+- cluster:
+    server: <direct-url-to-cluster>
+```
+
+## Issues logging in with chainctl
+
+Ensure that you're pointing to production. You can check the `chainctl` config.
+
+```sh
+chainctl config view 
+```
+
+The config should be pointing to the production environment, as in the next output.
+
+```
+platform:
+    api: https://console-api.enforce.dev
+    audience: https://console-api.enforce.dev
+    console: https://console.enforce.dev
+    issuer: https://issuer.enforce.dev
+    registry: https://cgr.dev
+    timestamp-authority: https://tsa.enforce.dev
+```
+
+For more guidance on working with the `chainctl` config, review our guide on [How to Manage `chainctl` Configuration](/chainguard/chainguard-enforce/manage-chainctl-config/).
+
+## Policies 
+
+If you're running into issues with policies or cluster image policies, this section has some tips for recovery.
```

Comment:
```suggestion
If you're running into issues with policies, this section has some tips for recovery.
```
I don't think they're different enough to present them as separate ideas here. Like, I think I could articulate the difference between "policies" and "cluster image policies" but I'm not sure it matters to users

Reply:
Ok! I was trying to be specific but I agree